mne/viz/raw.py

```diff
@@ -510,6 +510,7 @@ def _prepare_mne_browse_raw(params, title, bgcolor, color, bad_color, inds,
     """Helper for setting up the mne_browse_raw window."""
     import matplotlib.pyplot as plt
     import matplotlib as mpl
+    plt.ion()
```

Comment:
This is a `matplotlib` global state thing, and I don't think we should touch it unless we absolutely have to. I don't think users will expect us to change this setting. For example, sometimes I want plots to block, sometimes I don't, and `ion()`/`ioff()` currently accomplishes that, but this change will break it.

What warnings were you getting?